miniapps/meshing/mesh-optimizer.hpp

```diff
@@ -415,13 +415,23 @@ int material_id(int el_id, const GridFunction &g)
    double integral = 0.0;
    g.GetValues(el_id, ir, g_vals);
    ElementTransformation *Tr = fes->GetMesh()->GetElementTransformation(el_id);
-   for (int q = 0; q < ir.GetNPoints(); q++)
+   int approach = 1;
```

Comment:
Are changes in `mesh-optimizer` still needed? There aren't corresponding changes in `pmesh-optimizer`.